BE/LegalEntities/LEIEntities.rdf

```diff
@@ -367,14 +367,13 @@
 		<fibo-fnd-utl-av:adaptedFrom>GLEIF Level 2 Relationship Record (RR) Common Data Format (CDF), see https://www.gleif.org/en/about-lei/common-data-file-format/relationship-record-cdf-format#</fibo-fnd-utl-av:adaptedFrom>
 	</owl:NamedIndividual>
 	
-	<owl:ObjectProperty rdf:about="&fibo-be-le-lei;hasAddressOfLegalFormation">
+	<owl:ObjectProperty rdf:about="&fibo-be-le-lei;hasLegalAddress">
 		<rdfs:subPropertyOf rdf:resource="&fibo-be-le-fbo;hasRegisteredAddress"/>
-		<rdfs:label>has address of legal formation</rdfs:label>
+		<rdfs:label>has legal address</rdfs:label>
 		<rdfs:domain rdf:resource="&fibo-be-le-lp;LegalEntity"/>
-		<rdfs:range rdf:resource="&fibo-be-le-fbo;RegisteredAddress"/>
-		<skos:definition>indicates the address of legal formation for the entity, in the jurisdiction in which the entity is established, used for registration purposes with respect to obtaining an LEI</skos:definition>
+		<rdfs:range rdf:resource="&fibo-fnd-plc-adr;ConventionalStreetAddress"/>
+		<skos:definition>indicates the legal address for the entity, in the jurisdiction in which the entity is established, used for registration purposes with respect to obtaining an LEI</skos:definition>
```

Comment:
All formal organizations have a legal address whether or not they plan to, or are even eligible for, a LEI.

Reply:
True - it's in this ontology due to historical reasons only.